src/main/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessage.java

```diff
@@ -461,10 +365,49 @@ public static void setLocale(Locale locale) {
         }
     }
 
+    /** Clears the cache. */
+    public static void clearCache() {
+        BUNDLE_CACHE.clear();
+    }
+
     ////////////////////////////////////////////////////////////////////////////
     // Interface Comparable methods
     ////////////////////////////////////////////////////////////////////////////
```

Comment:
Should be just before `public int compareTo(LocalizedMessage other)`

Reply:
Done.